cmd/controller/diskstore.go

```diff
@@ -103,6 +103,7 @@ func (c *Controller) DeleteDisk(name string) {
 // and returns list of disk resources.
 func (c *Controller) ListDiskResource() (*apis.DiskList, error) {
 	label := NDMHostKey + "=" + c.HostName
+	label = label + "," + NDMManagedKey + "!=" + FalseString
```

Comment:
@shovan, test the upgrade scenario also. As When we will upgrade NDM, old disk CR won't have "ndm.io/managed" label and we are passing it in label selector.

Reply:
done.